_layouts/default.html

```diff
@@ -29,31 +29,31 @@
     <core-scaffold mode="waterfall-tall">
       <nav navigation>
         <paper-item label="Home">
-          <a href="//google.github.io/dagger/"></a>
+          <a href="./"></a>
         </paper-item>
         <paper-item label="User's Guide">
-          <a href="http://google.github.io/dagger/users-guide.html"></a>
+          <a href="users-guide"></a>
```

Comment:
Do we know this layout won't be used in a subdirectory? Should these look like `/dagger/users-guide` isntead?